plugins/gradle/gradle-plugin/src/main/java/org/wildfly/swarm/plugin/gradle/ThorntailExtension.java

```diff
@@ -60,6 +62,26 @@
 
     private boolean hollow = false;
 
+    private Integer debugPort = null;
+
+    private File stdoutFile;
```

Comment:
`stdout` or `stdOut`? Either way works. Please use the one that is more prevalent.

Reply:
Maven plugin has
```
    @Parameter(alias = "stdoutFile", property = "thorntail.stdout")
    public File stdoutFile;
```
so I would keep this notation.